build.gradle

```diff
@@ -44,8 +44,8 @@ dependencies {
     compile "io.dropwizard:dropwizard-auth:${dropwizardVersion}"
     compile "io.dropwizard:dropwizard-client:${dropwizardVersion}"
     compile "de.thomaskrille:dropwizard-template-config:1.4.0"
-    // compile "io.dropwizard:dropwizard-jdbi:${dropwizardVersion}"
-    // compile ':ojdbc6_g:'
+     compile "io.dropwizard:dropwizard-jdbi:${dropwizardVersion}"
+     compile ':ojdbc6_g:'
```

Comment:
Jenkins is going to provide "ojdbc6" upon deployment (without the "_g"). We should change this line to use "ojdbc6" instead. 

I created a PR to apply this change to the skeleton: https://github.com/osu-mist/web-api-skeleton/pull/62